### PR TITLE
[10.0] [ADD] sale_promotion_rule_delivery: Manage promotion rules when delivery addon is installed

### PR DESCRIPTION
**Pull request by lmignon**
- [X] includes #729

### CODE REVIEW

**Review by Cedric-Pigeon — APPROVED** [review]

File: sale_promotion_rule_delivery/models/sale_promotion_rule.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+class SalePromotionRule(models.Model):
+
+    _inherit = 'sale.promotion.rule'
+
+    def _get_lines_excluded_from_total_amount(self, order):
+        lines = super(
+            SalePromotionRule, self)._get_lines_excluded_from_total_amount(
+                order)
+        lines |= order.order_line.filtered(lambda l: l.is_delivery)
```
Comment (rousseldenis) [inline sale_promotion_rule_delivery/models/sale_promotion_rule.py]: You can replace by 
```suggestion
        lines |= order.order_line.filtered('is_delivery')
```

**Review by rousseldenis — APPROVED** [review]
Minor comment

**Comment by lmignon** [comment]
@rousseldenis comments fixed... can we move forward? ping @sebastienbeau @Cedric-Pigeon

**Comment by lmignon** [comment]
travis is :red_circle: due to https://github.com/OCA/sale-workflow/issues/817

**Comment by rafaelbn** [comment]
Please @lmignon could you rebase?

**Comment by lmignon** [comment]
@rafaelbn Rebase done. It seems that something is wrong with sale_exception. This one is not the only one affected by failing tests into sale_exception...

https://github.com/OCA/sale-workflow/pull/748#issuecomment-534017105

**Comment by rafaelbn** [comment]
@lmignon are you going to squash commits?

**Comment by lmignon** [comment]
@rafaelbn I would like to keep the history.

**Comment by lmignon** [comment]
@rafaelbn This one is now :green_heart:

**Review by rafaelbn — APPROVED** [review]

**Pull request merged**